additionalResolvers/course.ts

```diff
@@ -23,17 +23,9 @@ const resolvers: Resolvers = {
 
                 let chapters = root.chapters.elements.filter(chapter => {
                     // if chapter hasn't started yet, don't suggest it
-                    if(Date.parse(chapter.startDate) > currentTime) {
+                    if (Date.parse(chapter.startDate) > currentTime) {
                         return false;
                     }
-
-                    // if chapter doesn't have a suggested start date, suggest it
-                    if(chapter.suggestedStartDate === null) {
-                        return true;
-                    }
-
-                    // if chapter has a suggested start date and it's in the past, suggest it
-                    return Date.parse(chapter.suggestedStartDate) < currentTime;
                 });
```

Comment:
Don't you need to return true at the end of the filter function? (Or just Date.parse(chapter.startDate) > currentTime instead of the if block)